build/config/widget.config.js

```diff
@@ -36,6 +36,7 @@ const config = {
       failureThreshold: 0.001,
       failureThresholdType: 'percent', // pixel or percent
     },
+    assertNoLogError: false
```

Comment:
Needed to stop an error occurring here: https://github.com/Displayr/rhtmlBuildUtils/blob/9cced162f1da63d647ce87d26de2edf05993b50a/src/tasks/snapshot/assets/takeSnapshots.jest.test.js#L69

Reply:
I might have added this in the past. Probably OK disabling it for now since we have bigger priorities, and when we migrate to ngviz we will probably redo the tests.